platform/ui/src/components/snackbar/SnackbarContainer.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import SnackbarItem from './SnackbarItem';
+import './Snackbar.css';
+import { useSnackbarContext } from '../../../../viewer/src/context/SnackbarContext';
```

Comment:
The tricky thing, and this is less obvious in our monorepo setup, is that imports that cross package boundaries will cause the individual project builds to fail. If you need more clarification on this, let me know.